src/pages/api/users/addcard.js

```diff
@@ -0,0 +1,28 @@
+import dbConnect from "@/utils/dbConnect";
+import { modifyUserPaymentMethod } from "@/utils/dbFunctions"
+
+export default async function handler(req, res)
+{
+    const { method } = req;
+
+    await dbConnect();
+
+    switch (method)
+    {
+        case "PATCH":
+            try
+            {
+                await modifyUserPaymentMethod(req.body.userName, req.body.paymentMethods);
```

Comment:
While it's fine to do this in a school assignment, I would never (or at least very rarely) try to store card details myself. It's much too sensitive, and there are lots of external services that deal with this. I just wanted to point this out, even though it's not directly relevant to this assignment, because I want you to remember it for the future. 😊